holidays/countries/pitcairn_islands.py

```diff
@@ -0,0 +1,64 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import JAN, JUN
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class PitcairnIslands(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """
+    Pitcairn Islands holidays.
+
+    References:
+        - https://en.wikipedia.org/wiki/Public_holidays_in_Pitcairn_Islands
+    """
+
+    country = "PN"
+    default_language = "en"
```

Comment:
If you don't plan to support localization, you need to delete everything related to it (language attributes, `tr`, languages in README).